lib/retry.rb

```diff
@@ -17,4 +17,4 @@ def self.!(block, attempt=0)
 end
 
 class RetryableError < StandardError ; end
-class UnretryableError < StandardError ; end
\ No newline at end of file
+class UnretryableError < StandardError ; end
```

Comment:
I think so too.  And we should probably not be rescuing, logging, and then just otherwise swallowing the exception.  We should probably let it bubble all the way up.